media-sound/lmms/lmms-1.2.0.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="Cross-platform music production software"
+DESCRIPTION="LMMS is a free cross-platform alternative to commercial \
+programs like FL Studio®, which allow you to produce music with your \
+computer. This includes the creation of melodies and beats, the \
+synthesis and mixing of sounds, and arranging of samples. You can have fun \
+with your MIDI-keyboard and much more; all in a user-friendly and modern \
+interface."
+HOMEPAGE="http://lmms.io/"
+COPYRIGHT="2004-2019 Tobias Doerffel et al."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/LMMS/lmms/archive/v${portVersion/\~/-}.tar.gz"
+CHECKSUM_SHA256="0c1be741b39b20023a34f16bad043ad668d054a0d776cc1e083d80f9cc9cb6a7"
+SOURCE_DIR="lmms-${portVersion/\~/-}"
+srcGitRev2="a66360f09e91cca9c7a6a16162bff6c3bd1f7efb"
+SOURCE_URI_2="https://github.com/rampantpixels/rpmalloc/archive/$srcGitRev2.tar.gz"
+CHECKSUM_SHA256_2="1aa23b2395eacc0fd86f197f0c4dffeb69107f5e0b73c2eed5c6b86ed2bef2b7"
+SOURCE_DIR_2="rpmalloc-$srcGitRev2"
+#PATCHES="lmms-$portVersion.patchset"
+ADDITIONAL_FILES="LMMS.rdef"
+
+ARCHITECTURES="!x86_gcc2 !x86_64"
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	lmms$secondaryArchSuffix = $portVersion
+	app:lmms$secondaryArchSuffix
```

Comment:
Didn't check the build, but "app:lmms" (no need for secondaryArchSuffix normally) should use the same as the actual app "LMMS"?